SwiftStaticLibrary/SwiftStaticLibrary.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SwiftStaticLibrary.swift
+//  SwiftStaticLibrary
+//
+//  Created by Chad Landis on 8/7/17.
+//  Copyright © 2017 Chad Landis. All rights reserved.
+//
+
+import Foundation
+
+public struct SampleSwiftStructFromStaticLibrary {
+    public let name: String
+
+    public init(name: String) {
+        self.name = name
+    }
+
+    public func fooMethod() -> String {
+        return SampleObjcClassFromStaticLibrary().stringMethod1()
+    }
+}
+
+public class SampleSwiftClassFromStaticLibrary: NSObject {
```

Comment:
This Swift class should be accessible to Objective-C, but not sure how to make that happen.